packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -270,6 +279,16 @@ const DashboardHeader = ({
                             }}
                         />
                     )}
+                    {isScheduledDeliveriesModalOpen && dashboardUuid && (
+                        <DashboardSchedulersModal
+                            dashboardUuid={dashboardUuid}
+                            name={dashboardName}
+                            isOpen={isScheduledDeliveriesModalOpen}
```

Comment:
same as passing isOpen or it can be without `isScheduledDeliveriesModalOpen &&` if modal is lazy